virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc

```diff
@@ -0,0 +1,15 @@
+[id="virt-configuring-a-watchdog"]
+= Configuring a watchdog
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-a-watchdog
+
+toc::[]
+
+Expose a watchdog device on a virtual machine instance (VMI), alongside liveliness and readiness probes, to ensure the operating system is running properly.
+
+== Prerequisites
+
+* Kernel support for i6300esb inside the guest
+
+include::modules/virt-installing-a-watchdog.adoc[leveloffset=+1]
+include::modules/virt-defining-a-watchdog.adoc[leveloffset=+1]
```

Comment:
I think this should be the other way around: a user should first define the watchdog device, and only then configure it in the guest
```suggestion
include::modules/virt-defining-a-watchdog.adoc[leveloffset=+1]
include::modules/virt-installing-a-watchdog.adoc[leveloffset=+1]
```